src/Jackett.Common/Definitions/yggtorrent.yml

```diff
@@ -8,7 +8,7 @@ encoding: UTF-8
 followredirect: true
 requestDelay: 2
 links:
-  - https://www3.yggtorrent.cool/
+  - https://www.ygg.re/
```

Comment:
please copy the old domain to the bottom of the legacy section so that users ygg config will automatically be updated to the new domain when their Jackett gets the update.

Reply:
done 👍